packages/cli/src/services/project.service.ee.ts

```diff
@@ -198,20 +187,10 @@ export class ProjectService {
 	}
 
 	async updateProject(
-		name: string,
 		projectId: string,
-		icon?: { type: 'icon' | 'emoji'; value: string },
+		data: Pick<UpdateProjectDto, 'name' | 'icon'>,
 	): Promise<Project> {
-		const result = await this.projectRepository.update(
-			{
-				id: projectId,
-				type: 'team',
-			},
-			{
-				name,
-				icon,
-			},
-		);
+		const result = await this.projectRepository.update({ id: projectId, type: 'team' }, data);
```

Comment:
I see a risk here that the user of this `updateProject` might accidentally pass an object that contains other properties than `name` and `icon` that accidentally end up updated on the project. Maybe we could explicitly update only those props?

```suggestion
    const { name, icon } = data;
		const result = await this.projectRepository.update({ id: projectId, type: 'team' }, { name, icon });
```

Reply:
agreed. but that is a part of the next PR, where we move the user-management stuff our of this method/endpoint.